client/javascripts/components/markdown/markdown.tag

```diff
@@ -0,0 +1,19 @@
+import showdown from 'showdown'
+
+<markdown>
+    <div></div>
+
+    <script type="es6">
+        this.set = () => {
+            this.root.childNodes[0].innerHTML = convert(this.opts.content)
+        }
+        this.on('update', this.set)
+        this.on('mount', this.set)
+
+        var convert = (markdown) => {
```

Comment:
```javascript```
this.converter = new showdown.Converter()

this.convert = () => {
    this.root.childNodes[0].innerHTML = this.converter.makeHtml(this.opts.content)
}

this.on('update', this.convert)
this.on('mount', this.convert)
```